src/x-components/adapter.ts

```diff
@@ -1,30 +1,16 @@
 import { EmpathyAdapterBuilder } from '@empathy/search-adapter';
-import { Result } from '@empathy/search-types';
+import { customRequestMapper } from '@/adapters/demo-request-mapper';
+import { resultMapper } from '@/adapters/demo-result.mapper';
 
 export const adapter = new EmpathyAdapterBuilder()
```

Comment:
If we don't specify an `instance` here, the default value is `demo`. We are passing to the `SnippetConfig` in the `index.html` `platform` as the instance's default value if you don't set one via url. Should we set it here to `platform` or change it in the `index.html`? I'd just set it here as `platform` so we know to which "`instance`" this config belongs to.
```suggestion
export const adapter = new EmpathyAdapterBuilder()
  .setInstance('platform')
```

Reply:
We can move it here if you prefer, sure!